pkg/knuu/instance.go

```diff
@@ -596,24 +596,8 @@ func (i *Instance) Clone() (*Instance, error) {
 		return nil, fmt.Errorf("error generating k8s name for instance '%s': %w", i.name, err)
 	}
 	// Create a new instance with the same attributes as the original instance
-	return &Instance{
-		name:              i.name,
-		k8sName:           newK8sName,
-		imageName:         i.imageName,
-		state:             i.state,
-		instanceType:      i.instanceType,
-		kubernetesService: i.kubernetesService,
-		builderFactory:    i.builderFactory,
-		kubernetesPod:     i.kubernetesPod,
-		portsTCP:          i.portsTCP,
-		portsUDP:          i.portsUDP,
-		files:             i.files,
-		command:           i.command,
-		args:              i.args,
-		env:               i.env,
-		volumes:           i.volumes,
-		memoryRequest:     i.memoryRequest,
-		memoryLimit:       i.memoryLimit,
-		cpuRequest:        i.cpuRequest,
-	}, nil
+	ins := i.cloneWithSuffix("")
+	ins.k8sName = newK8sName
+	ins.instanceType = i.instanceType
```

Comment:
Instead of setting this here, this should be part of the `cloneWithSuffix` function. I just saw that I missed adding that to the struct creation there, which is a bug. Good catch! Could you remove the line here and add it to the struct creation in `cloneWithSuffix`?

Reply:
yes of course. i did it.